bot/exts/info/python_news.py

```diff
@@ -72,6 +73,11 @@ async def get_webhook_names(self) -> None:
             if mail["name"].split("@")[0] in constants.PythonNews.mail_lists:
                 self.webhook_names[mail["name"].split("@")[0]] = mail["display_name"]
 
+    @staticmethod
+    def escape_markdown(content: str) -> str:
+        """Escape the markdown underlines and spoilers."""
+        return re.sub(r"[_|]", lambda match: "\\" + match[0], content)
```

Comment:
Can we add `*` here too?
```suggestion
        return re.sub(r"[_*|]", lambda match: "\\" + match[0], content)
```

Reply:
I understand that it may capture honest markdown such as *italic*, but I would prefer we escape most markdown. If you look at the mailing list through the archive that the embed links you don't get this formatting so we might as well escape it as well?

Reply:
I can add it but I was asked to keep it there https://canary.discord.com/channels/267624335836053506/635950537262759947/873320120494211152

Reply:
Ah, right. In that case leave it, I'll mark this as resolved.